implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/DynatraceConfig.java

```diff
@@ -64,14 +67,32 @@ default String group() {
         return get(prefix() + ".group");
     }
 
+    /**
+     * Return the version of the target Dynatrace API.
+     *
+     * @return a {@link String} containing the version of the targeted Dynatrace API.
+     */
+    default String apiVersion() {
+        return getString(this, "apiVersion")
+                .map(x -> StringUtils.isEmpty(x) ? "v1" : x)
```

Comment:
```suggestion
                .map(val -> StringUtils.isEmpty(val) ? "v1" : val)
```
`x` looks a bit cryptic